share/ext/luajit/make-patch.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# The platform is just to be able to load it
+. ./version.sh
+
+# VERSION from build_luajit.h
+readonly PATCH_FILE=patch_${VERSION}
+
+ORIGINAL_REPO=git@github.com:LuaJIT/LuaJIT.git
+CHANGED_REPO=git@github.com:JCash/LuaJIT.git
```

Comment:
SInce creating and maintaining patches is hard, I'm now testing having an actual repo that I've made the relevant changes.

Then it should be easier to solve merge conflicts when I merge from upstream.
We may want to move this repo to Defold (as opposed to my personal account.

Reply:
This approach with our own fork feels much better and easier to maintain. 👍